### DISCUSSION TITLE
Polaris App/Update Set Picker

### DISCUSSION THREAD

**Discussion by mgrak** [arnoudkooi/SN-Utils#433] [Ideas]
In Polaris it does not show the current update set and or application without clicking the drop down menu.

Can you modify the DOM to have it always showing on header?

Good:
![image](https://github.com/arnoudkooi/ServiceNow-Utils/assets/37604982/376e8058-476a-4bfb-b184-8433d4d563f8)

Bad :(
![image](https://github.com/arnoudkooi/ServiceNow-Utils/assets/37604982/04ef79c4-1361-4563-a577-dd889a433d2e)

![image](https://github.com/arnoudkooi/ServiceNow-Utils/assets/37604982/00e2f2a6-3ddc-40ff-a90f-6a12ceadd529)

**Comment by arnoudkooi** [comment]
Check your settings....
![Google Chrome20230907 - 1806](https://github.com/arnoudkooi/ServiceNow-Utils/assets/840781/5eaa0e88-370a-47d7-a4c4-f90a6db4b69e)

**Reply by mgrak** [reply]
Oh nice! Thanks!
